src/components/Comments.jsx

```diff
@@ -1,9 +1,11 @@
 import { useEffect, useState } from "react"
 import { getComments } from "../api";
 import CommentItem from "./CommentItem";
+import CommentForm from "./CommentForm";
 
 const Comments = ({article_id}) => {
     const [comments, setComments] = useState([])
+    const [newComments, setNewComments] = useState([])
```

Comment:
You shouldn't need a separate array. Simply spread the current array and add to it or remove from it.

Reply:
done!! thanks so much 😄 🙌